content/Reducing_Power_Consumption.md

```diff
@@ -59,14 +59,21 @@ power standby mode. For example, a program may use this feature when it
 hasn't sensed keyboard input for a longer period (for example, when
 somebody forgot to turn off the Game Boy).
 
-Before invoking STOP, it might be required to disable Sound and Video
-manually (as well as IR-link port in CGB). Much like HALT, the STOP
-state is terminated by interrupt events. STOP is commonly terminated
-with a joypad interrupt.  A bug may keep STOP from terminating if STOP
-was invoked while a button was held.
-
-During STOP mode, the display will turn white, so avoid using it in your
-game's main loop.
+No licensed rom makes use of STOP outside of CGB speed switching.
+Special care need to be taken if you want to make use of the STOP
+instruction.
+
+On a DMG, disabling the LCD before invoking STOP, leaves the LCD enabled
+drawing a horizontal black line on the screen and damaging the hardware.
```

Comment:
Is it proven that it damages the hardware? AFAIK, it still hasn't been proven nor disproven

Reply:
It gives the same result as LCD off outside VBlank, which the nintendo manual mentioned as causing damage. Also, someone on discord did mention having semi-permanent damage after LCD off outside vblank. And it is quite clearly driving the LCD outside of specs.

Reply:
Nintendo's manual ([archive](https://ia803208.us.archive.org/9/items/GameBoyProgManVer1.1/GameBoyProgManVer1.1.pdf)) actually only mentions, on page 255, that:

> In early DMGs, a black horizontal line appears on the screen if the LCDC is stopped (LCDC register bit 7 ← 0) at any time other than during vertical blanking.

It also mentions, oddly enough, that LCDC should only be turned off when reading $9*1* or greater from `rLY`—not $90 as software typically does. Finally, this is only listed as *recommended*, whereas I would have expected it to be listed as "required" if it could damage hardware at all.

The same "programming cautions" section has an entry (page 257) about `stop`, but only notes that `rIE` should be set to 0 beforehand.

Reply:
Well, if you try it, it ignores the contrast setting, which is a pretty big red flag in my book. And even if the manual doesn't mention it, check on discord who tested this. (or find the old conversation)

I also rather be on the side of caution here.

Reply:
Me too, hence why I'd prefer if it was hinted as highly likely, but not guaranteed.